tests/test_config_flow.py

```
@@ -0,0 +1,1334 @@
+
+@pytest.mark.asyncio
+async def test_get_device_success():
+    """Test get_device returns device and client on successful auth."""
+    hass = MagicMock()
+    mock_client = MagicMock()
+    mock_client.connect = AsyncMock(return_value=AUTHN_SUCCESS)
+    mock_device = {"serialNumber": "SN123"}
+    mock_client.get_user_device_data = AsyncMock(return_value=mock_device)
+
+    with (
+        patch("custom_components.resmed_myair.config_flow.MyAirConfig") as mock_config,
```

Ruff (0.12.2)

Local variable `mock_config` is assigned to but never used

Remove assignment to unused variable `mock_config`

(F841)